pygerm/handler.py

```diff
@@ -22,28 +23,61 @@ def close(self):
 class BinaryGeRMHandler(HandlerBase):
     specs = {'BinaryGeRM'}
 
-    def __init__(self, fpath):
-        # TODO : don't save the raw data (here for debugging)
-        raw_data = np.fromfile(fpath, dtype='>u4')
-        # TODO : when simulated data comes in, verify this is correct
-        # endianness and correct for it, don't just raise error
+    def __init__(self, fpath, chunksize=None):
+        ''' Binary GeRM handler.
+
+            Parameters
+            ----------
+            chunksize : int, optional
+                if specified, this turns result into a dask array
+                This array is a lazy loaded array. To obtain the results
+                one must call the .compute() method.
+                This is useful when the data is too large to fit in memory.
+
+            Notes
+            -----
+            When using chunksize, use functools.partial, and functools.wraps
+            Example:
+                from functools import partial, wraps
+                bgermdask = wraps(BinaryGeRMHandler)(partial(BinaryGeRMHandler,
+                                                             chunksize=100000))
+                fhandler_init = bgermdask(filename)
+                res = fhandler_init['germ_ts']
+                # etc...
+        '''
+        raw_data = np.memmap(fpath, dtype='>u4')
+
+        # now the raw_data is a dask array, lazy loaded
+        if chunksize is not None:
+            raw_data = da.from_array(raw_data, chunks=chunksize)
```

Comment:
probably better to do this as 

```python
if chunksize is not None:
    data = da.from_array(raw_data, chunks=chunksize)
else:
    data = raw_data
```

so that below the first and last word access 'just works'.